Makefile

```diff
@@ -64,6 +64,13 @@ comprt: FORCE
 	@$(MAKE) runtime
 	@$(MAKE) modules
 
+notcompiler: FORCE
```

Comment:
Can we have this be a target that comprt calls to avoid the duplication (and avoid things becoming out of date if comprt changed?)

Reply:
That would change the possibilities for parallel builds across compiler vs comprt. I think that might be nice but I'd rather avoid it for the time being (as there might be a good reason why comprt is not already expressed as just a bunch of dependencies)